docs/MonthlyTrackedCompanies

```diff
@@ -0,0 +1,9 @@
+A Monthly Tracked Company (MTC) is the name of a company interacting with your open source project as surfaced by Scarf. When you add Scarf pixels to your pages or put a download behind the Scarf Gateway, Scarf’s IP enrichment will reveal the names of those companies. 
```

Comment:
Just want to be very precise here -- it's not the *name* of the company so much as the company's identity and all associate analytics that will be available to you for the rest of the month.